pkg/cmd/cli/cmd/rsync/forwarder.go

```diff
@@ -33,7 +35,8 @@ func (f *portForwarder) ForwardPorts(ports []string, stopChan <-chan struct{}) e
 	if err != nil {
 		return err
 	}
-	fw, err := portforward.New(dialer, ports, stopChan)
+	// TODO: Make os.Stdout/Stderr configurable
+	fw, err := portforward.New(dialer, ports, stopChan, os.Stdout, os.Stderr)
```

Comment:
same here, add as fields to `portForwarder`. there's only one place it's constructed, so add as args to `newPortForwarder(...)`. there's only one place that's called, and it has `RsyncOptions`, which already contains Our/ErrOut fields